### PR TITLE
Login Route Starting Point

### PR DESCRIPTION
**Pull request by debughug**
# General Purpose of Pull Request:
Using react router to display board and login

# Proposed Changes
Did a project restructure and kinda stopped there to reduce merge conflicts. 

# How to test?
Start up app and go to `/` and `/login`

# Add Applicable Screenshots
image
image

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -18,7 +18,4 @@ module.exports = {
     ecmaVersion: 2020, // Allows for the parsing of modern ECMAScript features
     sourceType: 'module', // Allows for the use of imports
   },
-  rules: {
-    'prettier/prettier': 'error',
-  },
-}
```
Comment (debughug) [inline .eslintrc.js:24]: Error felt aggressive so dropped it. Can revert if dispute.

File: src/App.js [hunk 2]
```diff
@@ -1,14 +1,23 @@
 import React from 'react';
-import { GameBoard } from './Components/GameBoard';
-
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import './Styles/_main.scss';
 
-function App() {
+import { TiCTacToe } from './Routes/TicTacToe';
+import { Login } from './Routes/Login';
```
Comment (debughug) [inline src/App.js:6]: Placed views in own route.

File: src/App.js [hunk 3]
```diff
@@ -1,14 +1,23 @@
...
+        <Switch>
+          <Route path="/login">
+            <Login />
+          </Route>
+          <Route path="/">
+            <TiCTacToe />
+          </Route>
+        </Switch>
+      </div>
+    </Router>
```
Comment (debughug) [inline src/App.js:21]: Showing route on path

File: src/Routes/Login/index.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+                type="text"
+                name="login"
+                id="login"
+                tabindex="1"
+                class="text-input"
+                autocorrect="off"
+                autocapitalize="off"
+              />
+            </fieldset>
+            <input class="button form-sub" type="submit" value="Sign In" tabindex="3"></input>
```
Comment (debughug) [inline src/Routes/Login/index.js:42]: Stub for now

File: src/Styles/abstracts/_variables.scss [hunk 5]
```diff
@@ -1,5 +1,11 @@
...
+$white-2: #fffafb;
+$black-2: #2b2c28;
 
-// Units
-$base-spacing: 25px;
+/* Units */
+$base-spacing: 20px;
+
+/* UI Colors */
+$input-background: #f3f3f4;
```
Comment (debughug) [inline src/Styles/abstracts/_variables.scss:11]: More colors

File: .eslintrc.js (hunk 1 above)
Comment (debughug) [reply .eslintrc.js:24]: i'll remove

**Review by christulin — COMMENTED** [review]
Looks good!

**Review by christulin — APPROVED** [review]

**Pull request merged**